### PR TITLE
docs: add swagger.json file location to FAQ

### PR DESCRIPTION
**Pull request by jolheiser**
This just adds a mention on how to get the `swagger.json` for an instance.

### CODE REVIEW

**Review by silverwind — APPROVED** [review]

File: docs/content/doc/help/faq.en-us.md [hunk 1]
```diff
@@ -136,7 +136,8 @@ All Gitea instances have the built-in API and there is no way to disable it comp
 You can, however, disable showing its documentation by setting `ENABLE_SWAGGER` to `false` in the `api` section of your `app.ini`.
 For more information, refer to Gitea's [API docs]({{< relref "doc/developers/api-usage.en-us.md" >}}).
 
-You can see the latest API (for example) on <https://try.gitea.io/api/swagger>.
+You can see the latest API (for example) on <https://try.gitea.io/api/swagger>.  
+You can also see an example of the `swagger.json` file at <https://try.gitea.io/swagger.v1.json>.
```
Comment (silverwind) [inline docs/content/doc/help/faq.en-us.md]:
```suggestion
You can see the latest API (for example) on <https://try.gitea.io/api/swagger>.

You can also see an example of the `swagger.json` file at <https://try.gitea.io/swagger.v1.json>.
```

Hard-break via trailing spaces is no longer possible using current markdownlint rules.

Comment (jolheiser) [reply docs/content/doc/help/faq.en-us.md]: *shakes fist at linter*

**Review by zeripath — APPROVED** [review]
I thought this was the standard location for this file...

**Comment by jolheiser** [comment]
> I thought this was the standard location for this file...

A cursory search suggested some people use `/swagger.json` among other things, so it looks like at the very least there is confusion in other projects as well.

**Comment by jolheiser** [comment]
🎺 🤖

**Comment by delvh** [comment]
> > I thought this was the standard location for this file...
> 
> A cursory search suggested some people use `/swagger.json` among other things, so it looks like at the very least there is confusion in other projects as well.

Take me for example.
TIL that we have a `swagger.json`.
I simply thought we didn't have one, given that none of the combinations I tried worked, and that there is none explicitly mentioned in the code (although I just found it now, `templates/swagger/v1_json.tmpl`).

**Pull request merged**